.github/workflows/tests.yml

```
@@ -92,7 +92,7 @@ jobs:
       - name: Set version
         run: echo '1.0' > VERSION
 
-      - uses: extractions/setup-just@95b912dc5d3ed106a72907f2f9b91e76d60bdb76  # 1.4
+      - uses: extractions/setup-just@69d82fb0233557aec017ef13706851d0694e0f1d  # 1.4
```

```suggestion
      - uses: extractions/setup-just@69d82fb0233557aec017ef13706851d0694e0f1d
```
version number does not match hash